### ISSUE TITLE
Investigate high quality speaker(s) and small circuit board with speaker amp, headphone amp and line out

### ISSUE THREAD

**Issue by radiance32** [yyzkevin/lxsidecar#21]
I'd like to investigate if we could add high-quality micro-speakers into the LXsidecar,
like the ones they use in modern tablets and smartphones.
Also, if I can find a small, ready made circuit board that contains an amp for this speaker (or speakers if we want stereo),
an amplified headphone amp, and a line out, so we can add a headphone jack on the front and a line out jack on the back of the LXsidecar.
I'd also investigate how to optimize the design of the housing so it improves the quality of the sound, mainly, using the housing as a speaker does, with a small port on the back (like many speakers have, so the drivers can push/pull sound through the front and back of the housing, for improved bass response...

**Comment by radiance32** [comment]
I think something like these would work:
https://nz.element14.com/pro-signal/abs-239-rc/speaker-8ohm-93db-0-7w/dp/2534981
nearly 1 watt of sound (700mah) and it's very compact, dimensions are in this spec PDF:
http://www.farnell.com/datasheets/2869785.pdf
Adding a pair of these to the LXsidecar in the front, behind some cutout horizontal lines would provide very high quality sound and quite loud too, like a modern tablet :)
They only cost aroun $4.50 each, which is pretty cheap...

**Comment by yyzkevin** [comment]
yeah this is no big deal, this is mostly a mechanical issue of how you want to fit it.

from analog side we need an amplifier anyway to drive headphones so it is going to be same idea.

for the topic on vogons about potential stand along  pcmcia card there may be some  more constraints to the amplifier because of profile of the card, but for our sidecar project we decide the form factor so we can do whatever we want

**Comment by radiance32** [comment]
I've found a tiny surface mount chip that's especially designed for driving tiny speakers like this,
we could add one or two on the sidecar PCB itself maybe ? 
https://www.ti.com/lit/gpn/lm4889
And then connect it to a speaker (or if we put 2, we can put 2 speakers and have stereo, but I don't think that's really necessary as both speakers will be mounted so close together that there won't be a stereo-field for your ears, so one of these and one of those tiny speakers I linked to for mono sound above should be plenty...
We'd have to add a separate op-amp for the headphone socket though as this is only a mono 1w power amp chip...

Terrence

**Comment by radiance32** [comment]
Ok,
I'll make a list of what we need and links to the PDFs containing the components technical details here,
example/reference schematics (or an additional PDF which contains one)...

Here we go:

[b]For the AD8397 opamp (the opamp to drive a set of headphones)[/b]

Here is the manufacturer's PDF with all the technical information (but no example/reference design/schematic in it...)
https://www.analog.com/media/en/technical-documentation/data-sheets/AD8397.pdf
However, here is a PDF containing the schematics of an evaluation board for op-amps like the AD8397, so you can use that as a reference design:
https://www.analog.com/media/en/technical-documentation/user-guides/UG-128.pdf (you could use this for any similarly packaged dual-opamps)
And, another one with a reference schematic: https://www.analog.com/media/en/technical-documentation/application-notes/AN-1429.pdf

SnapEDA page for the part:
https://www.snapeda.com/parts/AD8397/Analog%20Devices/view-part/?ref=search&t=AD8397


[b]For the LM4889 "boomer" 1 watt mono speaker amplifier SMD IC that we'd need to drive the micro-speaker in the lxsidecar[/b]

Manufacturer's PDF containing several reference schematics:
https://www.ti.com/lit/ds/symlink/lm4889.pdf?ts=1614212141955

SnapEDA page for part:
https://www.snapeda.com/parts/LM4889MM/Texas%20Instruments/view-part/?ref=search&t=LM4889


[b]For the 1 watt micro-speaker to be mounted inside the LXsidecar case[/b]

http://www.farnell.com/datasheets/2869785.pdf

SnapEDA page for part:
https://www.snapeda.com/parts/ABS-239-RC/Pro%20Signal/view-part/?ref=search&t=ABS-239-RC
(seems to be lacking a 2d and 3d model, but it says to click a button and they will create one for you... never used this site before so not sure about how this works and if you have to pay for it or something,
but it doesn't matter anyways as we're not going to mount the speaker on the PCB, it will be mounted into the lxsidecar's case and connected via wires to the PCB which contains the LM4889 amplifier IC...

There we go, if I find any others I will update this page, however I'm going to move this to the github issue page instead, and just leave this post here,
but we should continue discussing this on the github issue page...

Cheers,
Terrence
